src/Platform/Platforms.php

```diff
@@ -44,7 +44,7 @@ public function fromConnectionName(string $name): Platform
 
             return ($this->platformFactories[$platformName] ?? $this->fallbackPlatformFactory)($connection);
         } catch (Exception $e) {
-            throw new PlatformException($e->getMessage(), $e->getCode(), $e);
+            throw new PlatformException($e->getMessage(), (int)$e->getCode(), $e);
```

Comment:
Change due to phpstan complaints. Exception::__construct() accepts an int, but Exception::getCode() return mixed|int. Explicit casting still leads to an error (which is configured to be ignored), but feels cleaner.